Rakefile

```diff
@@ -1,7 +1,26 @@
 require 'rake/testtask'
 Rake::TestTask.new 'test' do |t|
+  if ENV['SOCKET_PATH'].nil?
+    sock_file = Dir.glob("#{__dir__}/**/redis.sock").first
+
+    if sock_file.nil?
+      puts '`SOCKET_PATH` environment variable required'
+      exit 1
+    end
+
+    ENV['SOCKET_PATH'] = sock_file
+    puts "SOCKET_PATH=#{sock_file}"
```

Comment:
I think we can do without that output line.

Reply:
I removed this step.